libultraship/libultraship/Window.cpp

```diff
@@ -199,6 +199,21 @@ extern "C" {
         return (char*)res->imageData;
     }
 
+    uint16_t ResourceMgr_LoadTexWidthByName(char* texPath) {
+        const auto res = static_cast<Ship::Texture*>(Ship::GlobalCtx2::GetInstance()->GetResourceManager()->LoadResource(texPath).get());
```

Comment:
this line might be useful as a helper function that gets inlined? it's duplicated in each of these functions.

Reply:
This is prime macro real estate.